chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -186,7 +126,7 @@
                 </entry>
                 <entry>
                     <key>email</key>
-                    <value></value>
+                    <value>amore@ainq.com☺aiyer@ainq.com</value>
```

Comment:
Will this overwrite whatever value exists in PROD? We might need to treat this job separately if so. Even if not, probably don't need that second email address in here

Reply:
This has been addressed with `<ignore-duplicates>` & `<overwrite-existing-data>` changes in jobs.xml